File: Store/StoreAPI/Models/Series.cs
using System;
using System.Collections.Generic;

namespace StoreAPI.Models;

public partial class Series
{
    public long SeriesId { get; set; }

    public string Title { get; set; } = null!;

    public string? Description { get; set; }

    public long FirmId { get; set; }

    public decimal Discount { get; set; }

    public virtual Firm Firm { get; set; } = null!;

    public virtual ICollection<Photo> Photos { get; set; } = new List<Photo>();

    public virtual ICollection<Product> Products { get; set; } = new List<Product>();
}

File: Store/StoreTgBot/src/models/series.py
from pydantic.v1 import BaseModel
from typing import List, Optional


class RequestCreateSeries(BaseModel):
    title: Optional[str] = None
    description: Optional[str] = None
    discount: Optional[float] = None
    firm_id: Optional[int] = None

    def __str__(self):
        return (
            f"title: {self.title}\n"
            f"description: {self.description}\n"
            f"discount: {self.discount}\n"
            f"firm_id: {self.firm_id}\n"
        )